moped-database/migrations/1743001305353_add_project_files_lookup/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE "public"."moped_file_types" ("id" serial NOT NULL, "name" text NOT NULL UNIQUE, "slug" text NOT NULL UNIQUE, PRIMARY KEY ("id"));
+
+COMMENT ON TABLE "public"."moped_file_types" IS 'lookup table of file types';
+INSERT INTO moped_file_types (id, name, slug) VALUES
+(1, 'Funding', 'funding'),
+(2, 'Plans', 'plans'),
+(3, 'Estimates', 'estimates'),
+(4, 'Other', 'other')
```

Comment:
I added the slug column because we have that pattern in other tables and I thought what if we needed to add a file type with more than one word, but now I am thinking this might be overkill since I am not using the slug anywere.